data/human/hails.txt

```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
 
+phrase "friendly civilian"
+	word
+		"Be careful of"
+		"A warning about"
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" "
+	word
+		"those space casinos. I blew"
+	word
+		" "
+	word
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" "
+	word
+		"million credits on "
+	word
+		"Martini"
+		"Thrall"
+		"Geminus"
+		"Hestia"
+		"Vinci"
+		"Vail"
+		"Calda"
+		"Pearl"
+		"Glory"
+		"Farseer"
+		"Pilot"
+		"Ada"
+		"Hermes"
+		"Mainsail"
+	word
+		" "
+	word
```

Comment:
Maybe add "playing" here? It makes it sound like each planet has their own version of each game. If that's what you want, it's fine, though.

Reply:
I don't think "I blew a million credits on Ada poker." Works as a sentence.